main.js

```diff
@@ -0,0 +1,34 @@
+const csv = require('csv-parser');
+const fs = require('fs');
+const results = []; //to store the result of the csv file here!
+
+function readPlain () {
+	fs.createReadStream('input.csv')
+	.pipe(csv({}))
+	.on('data', (data) => results.push(data))
+	.on('end', () => {
+		console.log(results);
+	}
+	);
+}
+
+function saveToFile (usersArr) {
+	var obj = {users: usersArr};		
+	var json = JSON.stringify(obj);
+	fs.writeFile('output.json', json, 'utf8', function(err){console.log(err);});
+}
+  
+function readJsonFile() {

```

Comment:
Same here, we should not hard code any value inside functions as mush as we can.